docs/appendices/release-notes/unreleased.rst

```diff
@@ -72,6 +72,9 @@ Changes
 - Users can now read tables within the ``pg_catalog`` schema without explicit
   ``DQL`` permission. They will only see records the user has privileges on.
 
+- Now accepts ``START TRANSACTION`` statements for Postgres compatibility, but
+  as CrateDB does not support transactions, this command has no effect.
```

Comment:
```suggestion
- Added support for the ``START TRANSACTION`` statement for Postgres compatibility, but
  as CrateDB does not support transactions, this command has no effect.
```

Reply:
Thanks, will make the change. Initially I have the same wording but thought it interfered with the second part of the sentence where it then says Crate does not support.